### PR TITLE
Zone Aware support documentation on README.md

### PR DESCRIPTION
**Pull request by lazerion**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +37,29 @@ or you can choose configuring AWS discovery using [Discovery SPI](http://docs.ha
 Starting with hazelcast-aws-1.2, it's strongly recommended to use Discovery SPI implementation as
 the former one will be deprecated.
 
+### Zone Aware Support
+
+Supported Hazelcast client version is `>= 3.8.6`
```
Comment (mesutcelik) [inline README.md]: Is this only for ZONE_AWARE support or in general hazelcast-client is fixed in AWS environment? cc: @sancar

File: README.md [hunk 2]
```diff
@@ -36,6 +37,29 @@ or you can choose configuring AWS discovery using [Discovery SPI](http://docs.ha
...
+
+As discovery services, these plugins put zone information to the Hazelcast member attributes map during the discovery process.
```
Comment (mesutcelik) [inline README.md]: What do you refer to when you use `those plugins`? This sentence is not very clear to me.

Comment (Serdaro) [reply README.md]: I think it should be as follows:
"As a discovery service, Hazelcast AWS plugin put the zone information into the Hazelcast's member attributes map during the discovery process. Please see the [Defining Member Attributes section](http://docs.hazelcast.org/docs/latest/manual/html-single/index.html#defining-member-attributes) to learn about the member attributes."

File: README.md (hunk 1 above)
Comment (Serdaro) [reply README.md]: Not to break the consistency with other documents:
"This support is available for Hazelcast Client 3.8.6 and newer releases.

File: README.md [hunk 3]
```diff
@@ -36,6 +37,29 @@ or you can choose configuring AWS discovery using [Discovery SPI](http://docs.ha
...
 the former one will be deprecated.
 
+### Zone Aware Support
+
+Supported Hazelcast client version is `>= 3.8.6`
+
+As discovery services, these plugins put zone information to the Hazelcast member attributes map during the discovery process. 
+Hazelcast creates the partition groups with respect to member attributes map entries that include AWS Availability Zone (AZ) information.
+
+When using `ZONE_AWARE` configuration, backups are created in the other AZ(s).
```
Comment (Serdaro) [inline README.md]: ....in the other AZs.

**Comment by Serdaro** [comment]
Reviewed.

**Comment by lazerion** [comment]
@mesutcelik @Serdaro updated the doc. please review.

**Pull request merged**